code/modules/power/supermatter/supermatter_delamination.dm

```diff
@@ -128,5 +160,70 @@
 		addtimer(CALLBACK(src, .proc/spawn_anomaly, anomaly_location, anomaly_to_spawn), current_spawn + extended_spawn)
 		current_spawn += next_spawn
 
+/**
+ * Callback for the anomalies to spawn after some time
+ */
 /datum/supermatter_delamination/proc/spawn_anomaly(location, type)
 	supermatter_anomaly_gen(location, type, has_changed_lifespan = FALSE)
+
+/**
+ * Setup for the cascade delamination
+ */
+/datum/supermatter_delamination/proc/start_universe_ending_cascade()
+	SSshuttle.registerHostileEnvironment(src)
+	SSshuttle.universal_cascade = TRUE
+	SSair.can_fire = FALSE
```

Comment:
Bro what the absolute fuck
If atmos can't run during it it shouldn't be done at all